ui/components/app/modals/import-token-exit-modal/import-token-exit-modal.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../../../store/actions';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import {
+  Text,
+  ButtonSecondary,
+  ButtonPrimary,
+  Box,
+} from '../../../component-library';
+import {
+  TextAlign,
+  FontWeight,
+  TextVariant,
+  Size,
+} from '../../../../helpers/constants/design-system';
+
+function mapStateToProps(state) {
+  return {
+    history: state.appState.modal.modalState.props.history,
+  };
+}
+
+function later(delay) {
+  return new Promise(function (resolve) {
+    setTimeout(resolve, delay);
+  });
+}
```

Comment:
I had  to use this trick because when i just call 
```
 dispatch(actions.clearPendingTokens());
 dispatch(actions.hideModal());
```
 In the click it was executing only the first line, and not executing the `history.push(DEFAULT_ROUTE);`